moor/lib/src/runtime/types/custom_type.dart

```diff
@@ -39,3 +39,30 @@ class EnumIndexConverter<T> extends TypeConverter<T, int> {
     return (value as dynamic)?.index as int;
   }
 }
+
+/// Enforces nnbd for TypeConverters
+abstract class NnbdTypeConverter<D, S> extends TypeConverter<D, S> {
```

Comment:
I'm not sure about the abbreviated "by default" in the name here, maybe something like `NonNullableTypeConverter`? I don't really like that either, I think the name should point out that nulls are "passed through". I don't have a better idea though :D

Reply:
I also struggled with what to name it (and keep it concise). I forgot that bd meant "by default" so it's probably the wrong name for it. Maybe NullEnforcedTypeConverter? Though that might be confusing to users

Reply:
To me, `NullEnforcedTypeConverter` sounds like `null` values are not allowed to enter the converter at all. Maybe `NullAwareTypeConverter`?

Reply:
Sounds good to me